test/master-playlist-controller.test.js

```diff
@@ -484,6 +484,27 @@ QUnit.test('updates the duration after switching playlists', function() {
               '16 bytes downloaded');
 });
 
+QUnit.test('removes request timeout when segment timesout on lowest rendition',
+function() {
+  this.masterPlaylistController.mediaSource.trigger('sourceopen');
+
+  // master
+  standardXHRResponse(this.requests.shift());
+  // media
+  standardXHRResponse(this.requests.shift());
+
+  this.masterPlaylistController.masterPlaylistLoader_.onLowestEnabledRendition = () => {
```

Comment:
You could set the initial bandwidth estimate to something really low to cause the loader to actually select the lowest rendition and skip this mocking.